client/public/js/components/App.js

```diff
@@ -35,6 +35,9 @@ var App = React.createClass({
               <li><a href={this.makeHref('events')}>Events</a></li>
               {/* This is the toggler for shepherd/sheep */}
               <li><a href={this.makeHref('toggle')} onClick={this._changeMode}>{this.state.mode === 'shepherd' ? 'Sheep' : 'Shepherd'}</a></li>
+              <li><a href="/#/">Home</a></li>
```

Comment:
remove the anchor tags for home and events since we have those in the lines above